src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -97,6 +97,8 @@ void initNetworkCommissioningThreadDriver(void)
 #endif
 }
 
+NetworkCommissioning::ThreadScanResponse * scanResult;
```

Comment:
The reasoning why this works here is that `NetworkCommissioning::ThreadScanResponse` type is known by all has the header is included. But the cluster and the implementation of it isn't if the example does not have a zap file with the cluster configured. I can add a few #ifndef _NO_NETWORK_COMMISSIONING_DRIVER_ over this while scan functionality but I don't think it will impact the code behaviour if I leave it.

Reply:
Ok, no need to change that then.